workstation-backend/workspace/models.py

```diff
@@ -3,11 +3,12 @@
 
 # Create your models here.
 
+
 class Workspace(models.Model):
-    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
     name = models.CharField(max_length=200)
 
 
 class Background(models.Model):
-    workspace_id = models.ForeignKey(Workspace, on_delete=models.CASCADE)
-    image_url = models.URLField()
+    workspace = models.OneToOneField(Workspace, on_delete=models.CASCADE)
```

Comment:
Are these one-to-one since each workspace can only belong to one user even though each user can have multiple workspaces?

Reply:
This is one-to-one because one workstation can only have one background